graphics/src/ColladaExporter_TEST.cc

```diff
@@ -193,6 +193,120 @@ TEST_F(ColladaExporter, ExportCordlessDrill)
   common::removeAll(pathOut + "/tmp");
 }
 
+/////////////////////////////////////////////////
+TEST_F(ColladaExporter, ExportMeshWithSubmeshes)
+{
+
+  std::string boxFilenameIn = std::string(PROJECT_SOURCE_PATH) +
+      "/test/data/box.dae";
```

Comment:
```suggestion
  std::string boxFilenameIn = common::joinPaths(PROJECT_SOURCE_PATH,
      "/test/data/box.dae");
```

Reply:
Still needed.

Reply:
It looks like this went through unresolved.

I also noticed this test [failing on CI](https://github.com/ignitionrobotics/ign-common/pull/132/checks?check_run_id=1477457840):

```
  [ RUN      ] ColladaExporter.ExportCordlessDrill
  unknown file: Failure
  C++ exception with description "basic_string::_M_construct null not valid" thrown in the test body.
  [  FAILED  ] ColladaExporter.ExportCordlessDrill (3 ms)
  [ RUN      ] ColladaExporter.ExportMeshWithSubmeshes
  unknown file: Failure
  C++ exception with description "basic_string::_M_construct null not valid" thrown in the test body.
  [  FAILED  ] ColladaExporter.ExportMeshWithSubmeshes (3 ms)
```

Reply:
Not sure why this didn't fail in the action here? 

Fix here: https://github.com/ignitionrobotics/ign-common/pull/133